openselery/selery_utils.py

```diff
@@ -0,0 +1,83 @@
+import time
+import dns.resolver
+
+
+
+class Connector(object):
+    def __init__(self):
+        super(Connector, self).__init__()
```

Comment:
The `Connector` class does not have any logic in it. What is it good for?

Reply:
as mentioned above

> the connectors classes were a good idea from the get go, i changed it so that this connector-plugin scheme is no mandatory and provides a proper underlying base class where you guys can put all the generic "connector" stuff in the future (if necessary)

one could do something usefull later with it, especially if mutliple additional (maybe even rival) connectors are used to do things. I just prepared that class so my intention with it is clear.

"can be removed safely"